src/components/transactions/SignedMessagesHelpLink/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box, SvgIcon, Typography } from '@mui/material'
+import InfoIcon from '@/public/images/notifications/info.svg'
+import ExternalLink from '@/components/common/ExternalLink'
+import { useAppSelector } from '@/store'
+import { selectSafeMessages } from '@/store/safeMessagesSlice'
+
+const SignedMessagesHelpLink = () => {
+  const safeMessages = useAppSelector(selectSafeMessages)
+  const safeMessagesCount = safeMessages.data?.results.length ?? 0
+
+  if (safeMessagesCount === 0) {
+    return null
+  }
+
+  return (
+    <Box display="flex" alignItems="center" gap={1}>
+      <SvgIcon component={InfoIcon} inheritViewBox color="border" fontSize="small" />
+      {/* TODO: Add link to help article */}
+      <ExternalLink noIcon href="#">
```

Comment:
Do we have the article yet?

Reply:
Not yet. I've made @johannesmoormann aware and have a reminder.